src/main/java/edu/kit/provideq/toolbox/QiskitProcessRunner.java

```diff
@@ -0,0 +1,36 @@
+package edu.kit.provideq.toolbox;
+
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+/**
+ * Process runner with output post-processing specifically for invoking GAMS.
+ */
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class QiskitProcessRunner extends ProcessRunner {
```

Comment:
I don't think this is qiskit-specific. Maybe call this `PythonProcessRunner`?

Reply:
Good point. The process runners should rather be environment-specific, not specific to their use case. GAMS is a bit of a mixed bag there, so there isn't a strict border.